docker/nugus_sim/run.py

```diff
@@ -100,6 +101,25 @@ def update_config_files(args: dict) -> None:
     webots_config["port"] = int(port)
     write_config("webots.yaml", webots_config)
 
+    # Configure logging to /robocup-logs if it exists
+    if os.path.exists("/robocup-logs"):
```

Comment:
Why check if it exists? Is this meant to be a mounted volume?

Reply:
Yep, it will be a mounted volume which may or may not be there